omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/MockTaskServiceWrapper.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.module.webservices.rest.web;
+
+import org.openmrs.module.webservices.helper.TaskServiceWrapper;
+import org.openmrs.scheduler.SchedulerException;
+import org.openmrs.scheduler.TaskDefinition;
+import java.util.ArrayList;
+import java.util.List;
+
+public class MockTaskServiceWrapper extends TaskServiceWrapper {
+	
+	public List<String> registeredTasks = new ArrayList<String>();
+	
+	public List<String> scheduledTasks = new ArrayList<String>();
+	
+	@Override
+	public void saveTaskDefinition(TaskDefinition task) {
+		registeredTasks.add(task.getName());
+	}
+	
+	@Override
+	public void unloadTask(TaskDefinition task) throws SchedulerException {
+		scheduledTasks.remove(task.getName());
+		registeredTasks.remove(task.getName());
+	}
+	
+	@Override
+	public void scheduleTask(TaskDefinition task) throws SchedulerException {
+		boolean contains = false;
+		for (String taskDef : scheduledTasks) {
```

Comment:
Why don't we just get rid of this class too??
Shouldn't this just call? Context.getSchedulerService().scheduleTask(task)

Reply:
This is the MockTaskServiceWrapper class, I am using this for testing purpose. 
So at that point, I haven't any registered tasks to the schedule by Context.getSchedulerService().scheduleTask(task). 

I followed the MockModuleWrapper Idea here to do the tests.

Reply:
Oh i see!
If it is only used for testing purposes and hence in src/test, then you are correct to have it stay. 👍